docs/source/nbconvert_library.ipynb

```diff
@@ -323,7 +323,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "You can use `Preprocessors` to accomplish the task at hand.  IPython has preprocessors built in which you can use.  One of them, the `ExtractOutputPreprocessor` is responsible for crawling the notebook, finding all of the figures, and putting them into the resources directory, as well as choosing the key (i.e. `filename_xx_y.extension`) that can replace the figure inside the template.\n",
+    "You can use `Preprocessors` to accomplish the task at hand.  Jupyter has preprocessors built in which you can use.  One of them, the `ExtractOutputPreprocessor` is responsible for crawling the notebook, finding all of the figures, and putting them into the resources directory, as well as choosing the key (i.e. `filename_xx_y.extension`) that can replace the figure inside the template.\n",
```

Comment:
I would say either no caps or `Nbconvert`, but neither is really perfect. For nbgrader, I typically use no caps and avoid putting it in places where it needs to be capitalized, but you can't *always* avoid that :-/